misc/old/svelte/NftsList.svelte

```diff
@@ -6,7 +6,7 @@
   import { getNetwork } from "lib/kconfig";
   // import {  nftListKey } from "lib/kconfig";
 
-  import Nft from "../Nft/Nft.svelte";
+  import Nft from "../../../svelte/components/Nft/Nft.svelte";
```

Comment:
not necessary, try not changing files when move to misc/old dir

Reply:
done